File: java-lab/src/test/java/ga/overfullstack/DateTimeLab.java
package ga.overfullstack;

import java.time.LocalDate;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

class DateTimeLab {
  @Test
  @DisplayName("Local Date Time format")
  void localDateTimeFormat() {
    final var ldt = LocalDate.now();
  }
}

File: kotlin-lab/src/test/kotlin/ga/overfullstack/DateTimeLab.kt
package ga.overfullstack

import kotlinx.datetime.Clock
import org.junit.jupiter.api.Test

class DateTimeLab {

  @Test
  fun dateTime() {
    println("Clock now: " + Clock.System.now())
    println("Epoch seconds: " + Clock.System.now().epochSeconds)
  }
}
